package.json

```diff
@@ -1,12 +1,18 @@
 {
   "name": "social-logos",
-  "version": "1.0.0",
+  "version": "1.0.1-alpha.2",
   "main": "build/index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/Automattic/social-logos"
   },
   "license": "GPL-2.0+",
+  "scripts": {
+    "build": "$(npm bin)/grunt --verbose"
```

Comment:
npm scripts automatically include `node_modules/.bin` in path so this can be just `grunt --verbose`